retroarch/retroarch.json

```
@@ -0,0 +1,36 @@
+{
+	"name": "retroarch",
+	"config-opts": [ "--enable-dbus" ],
+	"make-args": [ "GLOBAL_CONFIG_DIR=${FLATPAK_DEST}/etc" ],
+	"sources": [
+		{
+			"type": "git",
+			"url": "https://github.com/libretro/RetroArch.git",
+			"branch": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
```

```suggestion
			"commit": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
```

commit ain't a branch

---

Fixed